data-serving/data-service/schemas/cases.schema.json

```diff
@@ -257,21 +257,25 @@
           }
         }
       },
-      "source": {
-        "bsonType": "object",
-        "additionalProperties": false,
-        "properties": {
-          "_id": {
-            "bsonType": "objectId"
-          },
-          "id": {
-            "bsonType": "string"
-          },
-          "url": {
-            "bsonType": "string"
-          },
-          "other": {
-            "bsonType": "string"
+      "sources": {
+        "bsonType": "array",
+        "uniqueItems": true,
+        "items": {
+          "bsonType": "object",
+          "additionalProperties": false,
+          "properties": {
+            "_id": {
+              "bsonType": "objectId"
+            },
+            "id": {
```

Comment:
Can you remind me what the distinction between the two ID fields is?

Reply:
sure -- _id is auto-generated by mongodb on insert for sub-documents

id is a reference to a document in the `sources` collection. (if we don't want to use this, can remove later!)